service-front/web/main.scss

```diff
@@ -39,10 +39,22 @@ $govuk-assets-path: "../assets/";
   margin-bottom:10px;
 }
 
+.app-moj-banner-wide {
+  border: 2px solid !important;
+  padding:5px;
+  margin-bottom:10px;
+  min-width: 250px;
+  vertical-align: baseline !important;
+}
```

Comment:
There's some duplication here with `app-moj-banner`. Could the element have both classes and the wide just add `min-width`/`vertical-align`?

Should probably also use double-dash to indicate it's a modifier (i.e. `app-moj-banner--wide`)